lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -4,30 +4,33 @@ module ExternalHttpdConfiguration
       #
       # External Authentication Definitions
       #
+      COPY_BASE_DIR       = "/var/www/miq/system/COPY/"
+      TEMPLATE_BASE_DIR   = "/var/www/miq/system/TEMPLATE/"
+
+      PAM_CONFIG_DIR      = "/etc/pam.d/"
+      HTTPD_CONFIG_DIR    = "/etc/httpd/conf.d"
+
       IPA_COMMAND         = "/usr/bin/ipa"
       IPA_INSTALL_COMMAND = "/usr/sbin/ipa-client-install"
       IPA_GETKEYTAB       = "/usr/sbin/ipa-getkeytab"
 
-      PAM_MODULE          = "httpd-auth"
-      PAM_CONFIG          = "/etc/pam.d/#{PAM_MODULE}"
-      PAM_CONFIGURATION   = <<EOS
-auth    required pam_sss.so
-account required pam_sss.so
-EOS
       SSSD_CONFIG         = "/etc/sssd/sssd.conf"
       HTTP_KEYTAB         = "/etc/http.keytab"
 
-      EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
-      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
-      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
+      CONFIG_FILE = {
+        :pam_module    => "httpd-auth",
+        :external_auth => "cfme-external-auth.conf",
+        :remote_user   => "cfme-remote-user.conf"
+      }
+
+      TEMPLATE_FILE = CONFIG_FILE.merge(
```

Comment:
for erb style template files, I do need both names, when un-configuring, I remove the non erb names in target directory.